test/pollfd.jl

```diff
@@ -1,52 +1,90 @@
 # This file is a part of Julia. License is MIT: http://julialang.org/license
 
-@unix_only begin
```

Comment:
not a bad goal, but will need to account for different spellings/api of posix functions - unless libuv has wrappers for all of these, which I think is one of the things we've been meaning to do upstream?